client/xr/pico_face_tracker.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * WiVRn VR streaming
+ * Copyright (C) 2024  galister <galister@librevr.org>
+ * Copyright (C) 2025  Sapphire <imsapphire0@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "pico_face_tracker.h"
+#include "wivrn_packets.h"
+#include "xr/instance.h"
+#include "xr/session.h"
+#include <openxr/openxr.h>
+
+xr::pico_face_tracker::pico_face_tracker(instance & inst, session & s) :
+        s(s)
+{
+	xrStartEyeTrackingPICO = inst.get_proc<PFN_xrStartEyeTrackingPICO>("xrStartEyeTrackingPICO");
+	xrStopEyeTrackingPICO = inst.get_proc<PFN_xrStopEyeTrackingPICO>("xrStopEyeTrackingPICO");
+	xrSetTrackingModePICO = inst.get_proc<PFN_xrSetTrackingModePICO>("xrSetTrackingModePICO");
+	xrGetFaceTrackingDataPICO = inst.get_proc<PFN_xrGetFaceTrackingDataPICO>("xrGetFaceTrackingDataPICO");
+
+	CHECK_XR(xrStartEyeTrackingPICO(s));
+	CHECK_XR(xrSetTrackingModePICO(s, XR_TRACKING_MODE_FACE_BIT_PICO));
+}
+xr::pico_face_tracker::~pico_face_tracker()
+{
+	CHECK_XR(xrStopEyeTrackingPICO(s, XR_TRACKING_MODE_FACE_BIT_PICO));
```

Comment:
CHECK_XR throws in case of error, in the destructor we should just log if it fails.